src/components/tabs/tab/tab.d.ts

```diff
@@ -1,13 +1,19 @@
-import * as React from 'react';
+import * as React from "react";
 import { PaddingSpacingProps } from "../../../utils/helpers/options-helper";
 
+export interface TabContextProps {
+  setError: (childId: string, hasError: boolean) => void;
```

Comment:
These should be optional I think

Reply:
as in previous components - context props